_data/projects.yml

```diff
@@ -724,3 +724,11 @@
   upforgrabs:
     name: help out!
     link: https://github.com/kellabyte/Haywire/labels/help%20out!
+    
+- name: .NET Core
+  desc: The class libraries for the next version of the .NET Framework
```

Comment:
That's a bit misleading as .NET Core is a self-contained stack. Some of the API will be ported to the .NET Framework, some will not. What about: `Contains the foundational libraries that make up the .NET Core development stack`